Added .opi fies packaging

oksanagit opened the pull request:
The folder and tgz files are stored in DEPLOYMENTS.

jwlodek commented on installSynApps/Driver/packager_driver.py:
```diff
@@ -397,3 +397,37 @@ def create_package(self, filename):
+        0  - if suceeded
+        -1 - if error
+        """
+        opi_dir = str(self.output_location + '/opis')
```
You can probably remove the `opi_dir` parameter for the method since you define it based on `self.output_location` here.

jwlodek commented on installSynApps/Driver/packager_driver.py:
```diff
@@ -397,3 +397,37 @@ def create_package(self, filename):
+                        print("Can't copy " + file_name + ' to ' + opi_dir)
+                        sys.exit(1)
+        #opis_bundle_name = self.create_bundle_name('opis.tgz')
+        out = subprocess.call(['tar', 'czf', 'DEPLOYMENTS/opis.tgz', '-C', 'DEPLOYMENTS/opis', '.'])
```
You may want to use `os.path.join(self.output_location, 'opis.tgz')` and `opi_dir` for the paths in the `tar` command, since if the user defines a different ouput location, the folder won't be placed in `DEPLOYMENTS/opis`, but in `self.output_location + '/opis'` as defined above.

oksanagit commented:
Please take a look. I addressed your comments. However, if we create ad_opis, script fails the second time, which could be OK for now. There is always room for improvements.

jwlodek commented:
Looks good, I merged the changes in. We can always work on improvements in the future as you said. We still should set up some cameras with these screens, so we know how the setup process would work with them.
